docker/php.Dockerfile

```diff
@@ -85,17 +82,17 @@ COPY composer.json composer.json
 COPY composer.lock composer.lock
 
 ARG COMPOSER_AUTH
-COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
+COPY --from=composer/composer:latest-bin /composer /usr/bin/composer
 RUN composer install --no-progress --ignore-platform-reqs --no-dev --prefer-dist --optimize-autoloader --no-interaction
 
 #============================================
 # COMMAND LINE INTERFACE
 #============================================
-FROM php:8.1.6-cli-alpine3.16 as cli
+FROM php:8.1.12-cli-alpine3.16 as cli
 
 # https://blog.packagecloud.io/eng/2017/02/21/set-environment-variable-save-thousands-of-system-calls/
-ENV TZ=:/etc/localtime
-ENV PHP_ENV=development
+ENV TZ=:UTC
+ENV PHP_ENV=dev
```

Comment:
@okiedork should this be kept as "development"?

Reply:
`dev`.